docs/source/guides/VTFModding.rst

```diff
@@ -9,64 +9,64 @@ VTF, short for "Valve Texture Format", is a texture type used by valve in the so
+
+	"UnlitTwoTexture"
+	{
+
+		"$surfaceprop" "metal"
+		"$basetexture" ""
+		"$texture2" ""
+		"$bumpmap" ""	
+		"$allowoverbright" "1"
+		"$vertexcolor" 1
+		"$vertexalpha" 1	
+		"$decal" "1"
+		"$model" 1
+		"$nocull" "1"
+	}
+
+When we use vtf textures, we can only use the albedo and normal `maps <https://titanfall-skin-group.gitbook.io/titanfall-2-skin-creation/ms/genral-information/texture-maps>`__. Fire up `VTFEdit <https://nemstools.github.io/pages/VTFLib-Download.html>`__ and hit file, import, and grab your texture. then file, save as, and save it in the same folder as your .vmt. In your vmt, put the path to your texture in the parentheces after `"$basetexture"`, treating models as root. So i would put, `models\weapons_r2\vtfkraber\kraber_col`. Then do the same for your normal map, but when you import it, pick volume texture instead of animated texture. In `"$bumpmap"` put the path to your normal texture. Now create another vtf with literally any image. Put its path in `"$texture2"`. As far as i know, this is neccesary even though the texture isnt used. Your root folder should look somewhat like this::
+
+	root
+	├─ materials
+	│  └─ models
+	│     └─ weapons_r2
+	│        └─ vtfkraber
+	│           └─ vtfkraber.vmt
+	├─ models
```

Comment:
```suggestion
	└─ models
```

Should probably not have a dangling edge unless that's intentional? :P

Reply:
I did do this in the github editor ~~while at work~~ so